src/components/Form/DateRange/DateRange.scss

```diff
@@ -13,6 +13,13 @@
     width: 5rem;
     text-align: center;
     padding-top: 7rem;
+
+    &:before {
+      content: '\f30b';
+      font-family: 'Font Awesome 5 free';
```

Comment:
Is there a reason why the `font-family` needs to be `Font Awesome 5 free` and not just `font-family: FontAwesome`?

I'm not familiar with `'Font Awesome 5 free'

Reply:
Font Awesome 5 Free is the updated font name that Font Awesome uses to declare which icon set to use. There is now a "free" version and a paid version so they separated the fonts into two categories. We are using the free version.